requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+importlib
+setuptools>25,<=38
```

Comment:
I used `setuptools<38` to get `make rpm` to work with Python 2.6 (see [here](https://github.com/cms-gem-daq-project/gem-plotting-tools/issues/107#issuecomment-394319947)). It wasn't working with `<39`.

Reply:
Do you know which version got pulled in?